test/unit/model/virtualbox-test.js

```diff
@@ -240,10 +241,17 @@ describe('Virtualbox installer', function() {
       }
     });
 
+    it('should able to check file were extracted to temp', function(done) {
+      sandbox.stub(child_process, 'execFile').yields(null, "Files were extracted to temp", "");
+      item2.setInstallComplete();
+      item2.thenInstall(installer);
+      installer.install(fakeProgress, success, failure);
```

Comment:
call installer.installAfterRequirements() and you would not need to create jdk installer at all.